cli/azd/grpc/proto/compose.proto

```diff
@@ -69,6 +70,7 @@ message ComposedResource {
   string type = 2;
   bytes config = 3;
   repeated string uses = 4;
+  string resource_id = 5;
```

Comment:
Since the `ComposedResource` already contains a `ResourceId`, do we really need to modify the `AddResourceRequest` ?